.gitignore

```
@@ -12,7 +12,7 @@ MCprep_addon/MCprep_resources/resourcepacks/mcprep_default/materials.blend1
 mcprep_venv_*
 .cache
 .python-version
-venv/
+.venv/
```

Maybe we should just do both, since our docs show quick start where venv isn't hidden? (though I do agree, hidden is more standard)

```suggestion
venv/
.venv/
```